packages/good-design/src/theme/hoc/withMultiTranslations.tsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { Trans } from "@lingui/react";
+
+/**
+ * A HOC that wraps a component with translations
+ * mainly used in any context where you cannot use <Trans> (e.g. in fragmented modals)
```

Comment:
why cant you use trans? what do you mean by fragmented modals?

Reply:
Similiar to `React.Fragment` the modals don't live/exist in the regular dom tree as they are placed inside a ReactPortal.
(We have explicit fragments, I believe this was done for web compatibility.

That makes them not wrapped by i18n provider, resulting in a error: You need to wrap with provider to use the trans component. (You can confirm this behavior by looking at the components tree, from the react dev tools)

this hoc renders the translated components before they get passed down to the modals.

Reply:
see my comment regarding location of the language provider

Reply:
fixed here: https://github.com/GoodDollar/GoodWeb3-Mono/pull/171/commits/ec20010257c4f45a3ce3de7877e9cbba36454fc1